lock-api/src/test/java/com/palantir/lock/HeldLocksTokenTest.java

```diff
@@ -0,0 +1,31 @@
+package com.palantir.lock;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableSortedMap;
+
+public class HeldLocksTokenTest {
+	@Test
+	public void testSerde() throws IOException {
```

Comment:
not for this PR, but more broadly we should have serialization/deserialization tests for all exposed API